src/js/browser-plugin.js

```diff
@@ -69,66 +56,81 @@ const start = () => {
 };
 
 const newTweetCallback = (tweetInfo) => {
-  
+
   tweetData = tweetInfo;
+  dom = tweetInfo.domObject;
   if (tweetInfo.links.length > 0) {
-    const score = {};
 
-    if (usersCache.hasOwnProperty(tweetInfo.username)) {
-      score.username = tweetInfo.username;
-      score.misinformationScore = usersCache[tweetInfo.username];
+    if (usersCache[tweetInfo.id] == null) usersCache[tweetInfo.id] = false; 
```

Comment:
Isn't the linter (JSHint) marking this as bad code style?

Reply:
Yes, i forgot to pass the hinter (other error were showing too).